src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Steps/AddPackagesStep.cs

```diff
@@ -13,7 +14,7 @@ public class AddPackagesStep : ScaffoldStep
     /// <summary>
     /// Gets or sets the list of package names to add.
     /// </summary>
-    public required IList<string> PackageNames { get; set; }
+    public required IList<Package> Packages { get; set; }
```

Comment:
Does this have to be a mutable `IList<>`, vs say `IReadOnlyList<>` or `ImmutableArray<>`?